server-postgres/changeDatabase.xsl

```diff
@@ -8,7 +8,7 @@
 
     <xsl:template match="//ds:subsystem/ds:datasources/ds:datasource[@jndi-name='java:jboss/datasources/KeycloakDS']">
         <ds:datasource jndi-name="java:jboss/datasources/KeycloakDS" enabled="true" use-java-context="true" pool-name="KeycloakDS" use-ccm="true">
-            <ds:connection-url>jdbc:postgresql://${env.POSTGRES_PORT_5432_TCP_ADDR}:${env.POSTGRES_PORT_5432_TCP_PORT:5432}/${env.POSTGRES_DATABASE:keycloak}</ds:connection-url>
+            <ds:connection-url>jdbc:postgresql://${env.POSTGRES_HOST}:${env.POSTGRES_PORT:5432}/${env.POSTGRES_DATABASE:keycloak}</ds:connection-url>
```

Comment:
The above is an example of the env vars auto-generated when using docker linking :( Converting to compose allows us to clean them up :)